charts/operator-wandb/values.yaml

```diff
@@ -102,6 +102,12 @@ global:
     brokerPort: 9092
     runUpdatesShadowTopic: ""
 
+  metadataStoreOverflow:
+    enabled: false
```

Comment:
So the idea is that we'll override this with channels?

Reply:
Yes